src/components/app-configure/AppUserLimits.vue

```diff
@@ -149,6 +163,8 @@ export default {
       value: 2,
       unit: "MB",
     });
+    let storageError = ref(false),
+      bandwidthError = ref(false);
```

Comment:
Prefix boolean variables with either `is`, `has` or `show`.

```suggestion
    let showStorageError = ref(false),
      showBandwidthError = ref(false);
```

Reply:
Yeah makes sense, I'll make the changes